presto-rcfile/src/test/java/io/prestosql/rcfile/RcFileTester.java

```diff
@@ -1121,6 +1122,7 @@ private static Properties createTableProperties(String name, String type)
         private final File tempDir;
         private final File file;
 
+        @SuppressWarnings("ResultOfMethodCallIgnored")
```

Comment:
wrap `tempDir.mkdirs()` below in `checkState( .. )` instead

Reply:
Actually, this `mkdirs()` is useless since `createTempDir()` will create it. Removed.